### DISCUSSION TITLE
On the proposal to hard fork airdrop tokens into a Handshake Foundation Multisig

### DISCUSSION THREAD

**Discussion by evbots** [handshake-org/HIPs#58] [General]
At a high level, we need to reduce supply and increase demand. Both matter, and they reinforce each other.

### Supply

Massive supply bombs hiding in the protocol (the current state of the world) increase uncertainty and reduce the ability for investors to confidently model scenarios. Demand is less likely when there is so much uncertainty around supply. A big pot of HNS controlled by a foundation has similar characteristics, even less desirable than the current situation.

### Demand

 External demand is needed. Browser integrations will drive a lot of that. Additionally, the protocol itself can drive demand for HNS ("internal demand"). We already see this at work with name auctions. Users demand HNS for auctions, then that HNS gets burned. There is also demand to pay transaction fees to miners, but that is roughly equally counteracted by sell pressure from miners, so we can exclude that. We can increase internal demand _now_. This can be a strong economic foundation that future external demand (browser integrations) can build on.

**How do we increase internal demand**? The simplest way to get there is boost the burn mechanism that HNS already has: name auctions. How can that happen? By lowering the RENEW limit per block. This basically means: some names are forced to expire -> go up again for auction -> more HNS burn. This demand is net positive.

There is also an effect of RENEW transaction fees becoming more competitive and expensive. This demand is net zero due to sell pressure from miners, but it does increase the velocity of money, which may increase liquidity in markets.

**The TL;DR:** Constrict supply and make it predictable. Give the market some concrete reasons to buy HNS to increase demand.

### A foundation multisig

I've seen the proposals to fork the airdrop into a multisig to pay for integrations. I think we could probably get some mileage out of that, but I don't think it will yield long term results. Many potentially integrators already have a token allocation and they simply don't care (cloudflare/icann/etc). Or they do claim it and immediately dump it (we know who i'm talking about). My prediction is that even if you are able to scrounge up enough USD to pay a major browser to work on using HNS, they'll hold the feature hostage annually and then deprecate it when nobody pays.

The best reasons for other apps to integrate HNS is a thriving ecosystem. I have other ideas for that, but since this root zone is a standalone blockchain, the demand for the token and names has to be brought in balance.

🙏 Thank you for reading 🙏

**Comment by nickorlabs** [comment]
So I think we will have many options and thoughts on the table for the hard fork.  I have been suggesting the community discuss the various options; along with discussing the soft fork and ensuring the community consensus is implemented in the change.

I am going to have a space/ or something of the sorts 6/19 or 6/20.  I hope you can attend.

@SaveTheDoctor
